test/integration/model/consent.test.ts

```diff
@@ -0,0 +1,176 @@
+import Knex from 'knex'
```

Comment:
do we need consent model & tests?

Reply:
relevant to https://github.com/mojaloop/als-consent-oracle/pull/3#discussion_r560924716